### ISSUE TITLE
Training loss gets into Equilibrium early, then rises while downstream accuracy also degrades on Sentinel-2 temporal pretraining

### ISSUE THREAD

**Issue by gajeshladhar** [facebookresearch/vjepa2#140]
Hi V-JEPA team,

I am seeing a training behavior that looks unstable / counterintuitive and wanted to check whether this is expected or if I might be missing something in the setup.

**Setup:**
```
Model: V-JEPA2.1 pretraining
CKPT : started from pretrained 2.1 (ViT-L/16)
Hardware: 1x H100
Training time: ~50 hours
Data: custom cloud-free Sentinel-2 dataset
Input: 12 bands, temporal sequence of 16 frames
```

**Configs :** 
```
m (ema) : 0.99925 (constant) 
lr : 5e-5 (constant)
lambda_context : 0.5  (constant) 
batch_size : 16
num_frames :  16
crop_size :  224
patch_size :  16
tublet_size :  1
cfgs_mask :  [
    {
        "spatial_scale":     (0.15, 0.15),
        "temporal_scale":    (1.0, 1.0),
        "aspect_ratio":      (0.75, 1.5),
        "num_blocks":        8,
        "max_temporal_keep": 1.0,
    },
    {
        "spatial_scale":     (0.7, 0.7),
        "temporal_scale":    (1.0, 1.0),
        "aspect_ratio":      (0.75, 1.5),
        "num_blocks":        2,
        "max_temporal_keep": 1.0,
    },
]
```
**Observed behavior:**

In the first ~2–3 hours, training loss drops from ~0.60 to ~0.30
For the next ~15 hours, loss stays around ~0.30
After that, loss slowly starts increasing and eventually reaches ~0.45 over many hours

**Downstream behavior:**

Downstream accuracy improves during the early stage of training
It goes from ~75% to ~86% in roughly the first ~15 hours
After that, downstream accuracy starts degrading and drops from ~86% to ~82% with more pretraining

**So after some point, both happen together:**

1. pretraining loss starts drifting upward
2. downstream accuracy starts drifting downward

**This makes me wonder:**

1. Could this indicate EMA / target encoder dynamics issues, masking setup issues, predictor mismatch, or optimization instability?
2. Are there any recommended diagnostics to check for this (teacher/student feature norms, variance, cosine similarity, predictor drift, masking ratio sensitivity, LR / WD schedule, etc.)?

I am attaching illustrative plots of the trend I observed.
Thanks — any guidance would be really helpful.

<img width="1980" height="1077" alt="Image" src="https://github.com/user-attachments/assets/d8ceb6e8-6778-46d3-b2d0-5e5404a97c0d" />
